recipe/meta.yaml

```diff
@@ -25,23 +25,52 @@ requirements:
   host:
     - ncurses                # [not win]
     - readline               # [not win]
-  run:
-    - ncurses                # [not win]
-    - readline               # [not win]
 
-test:
-  commands:
-    - sqlite3 --version
-    - if not exist %PREFIX%\\Library\bin\sqlite3.dll exit 1    # [win]
-    - if not exist %PREFIX%\\Library\lib\sqlite3.lib exit 1    # [win]
-    - test -f $PREFIX/lib/libsqlite3.dylib                     # [osx]
-    - test -f $PREFIX/lib/libsqlite3.so                        # [linux]
-    - if not exist %PREFIX%\\Library\include\sqlite3.h exit 1  # [win]
-    - test -f $PREFIX/include/sqlite3.h                        # [not win]
+
+outputs:
+  - name: libsqlite
+    files:
+      - include/sqlite3*          # [not win]
+      - share/man/man1/sqlite3*   # [not win]
+      - lib/libsqlite3*           # [not win]
+      - lib/pkgconfig/sqlite3.pc  # [not win]
```

Comment:
@jakirkham I feel like all these lines are just going to cause future pain for packagers. Is there a way to simplify things?